introduction.md

```diff
@@ -1,3 +1,10 @@
 Hello my Name is Markus.
 Nice to meet you.
 My favourite CodeSchool Path is JavaScript.
+DeadlySkills:
+* JavaScript
+* Cordova
+* PHP
+* C++
+* Java
+* a little Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?